tests/NonPhpFile/NetteDINeonMethodCallRenamer/Fixture/fixture.neon

```diff
@@ -0,0 +1,25 @@
+services:
+    firstService:
+        factory: Rector\Nette\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\FirstService
+        setup:
+            - add('key1', 'value1')
+            - add('key2', 'value2')
+
+    -
+        class: Rector\Nette\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\SecondService('service-name')
+        setup:
+            - add('first-key', 'first-value')
+            - add('second-key', 'second-value')
+-----
+services:
+    firstService:
+        factory: Rector\Nette\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\FirstService
+        setup:
+            - add('key1', 'value1')
+            - add('key2', 'value2')
+
+    -
+        class: Rector\Nette\Tests\NonPhpFile\NetteDINeonMethodCallRenamer\Source\SecondService('service-name')
+        setup:
+            - addConfig('first-key', 'first-value')
+            - addConfig('second-key', 'second-value')
```

Comment:
Nice :+1:

Reply:
Could you split these to 2 fixtures - factory & class? Just to make test as isolated as possible, for future debugging

Reply:
done